html/app.html

```diff
@@ -20,7 +20,110 @@
+              <th>Basic</th>
+              <th>Advance</th>
+              <th>Enterprise</th>
+            </tr>
+            <tr>
+              <td>10+ PAGES</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>REPONSIVE DEVELOPMENT</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>CMS IMPLEMENTATION</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>E-COMMENCE</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>MOBILE APP</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>EXTENDING LIMITS</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>DESIGN COMPLEXITY</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+            <tr>
+              <td>OWN ANALYTICS PLATFORM</td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+          </table>
+
+          <ul>
+            <li>Price</li>
+            <li><img src="../img/pricing/dollar1.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar2.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar3.png" alt="" /></li>
+          </ul>
+
+          <button>Get Started</button>
+          <button>Get Started</button>
+          <button>Get Started</button>
```

Comment:
Button does not have a type attribute. Something like <button type="button"> Get Started </button>. Also this should be in the tfoot too.